pkg/asset/agent/image/agentpxefiles.go

```diff
@@ -186,10 +186,11 @@ func (a *AgentPXEFiles) copy(filepath string, src io.Reader) error {
 }
 
 func (a *AgentPXEFiles) createiPXEScript(cpuArch, ipxeBaseURL, pxeAssetsFullPath string) error {
-	iPXEScriptTemplate := `#!ipxe 
-initrd --name initrd %s/%s 
-kernel %s/%s  initrd=initrd coreos.live.rootfs_url=%s/%s ignition.firstboot ignition.platform.id=metal 
-boot`
+	iPXEScriptTemplate := `#!ipxe
+initrd --name initrd %s/%s
+kernel %s/%s initrd=initrd coreos.live.rootfs_url=%s/%s ignition.firstboot ignition.platform.id=metal
+boot
+`
```

Comment:
This is updating changes introduced in the previous commit and should be squashed.